lib/chronik/projection.ex

```diff
@@ -193,7 +193,8 @@ defmodule Chronik.Projection do
   # Try to catch up to a future version coming from the PubSub
   # by fetching missing events from the Store.
   defp catch_up(version, projection_state, projection, store) do
-    case store.fetch(version) do
+    from = if version == :empty do :all else version end
```

Comment:
Probably this fits better in an onliner: `if version == :empty, do: :all, else: version`